.github/workflows/pull_request.yml

```diff
@@ -99,50 +109,18 @@ jobs:
           echo "KUBECONFIG=$HOME/.kube/config" >> $GITHUB_ENV
           echo "OPENEBS_NAMESPACE=openebs" >> $GITHUB_ENV
 
-      - name: Build images locally
-        run: make zfs-driver-image || exit 1;
```

Comment:
It feels like this image that is being used in the CI tests. Are we building from the local changes?

Reply:
Yes, we build locally and test because this is a PR, there is no push happening here.

Reply:
Oh, I see, this one is moved to line 87. I thought it's being removed.